index.js

```diff
@@ -343,7 +345,7 @@ export const createHandler = async ({ logger, pgPool, signerAddresses }) => {
       .catch(err => errorHandler(res, err, logger))
       .then(() => {
         logger.request(
-          `${req.method} ${req.url} ${res.statusCode} (${new Date() - start}ms)`
+          `${req.method} ${req.url} ${res.statusCode} (${new Date().getTime() - start.getTime()}ms)`
```

Comment:
I propose a different fix:

```js
const start = Date.now()
// ...
`${req.method} ${req.url} ${res.statusCode} (${Date.now() - start}ms)`
```

What do you think about this option - is it an improvement or not?

Reply:
I had initially implemented it that way in https://github.com/CheckerNetwork/spark-api/pull/535 but after [review from Julian](https://github.com/CheckerNetwork/spark-api/pull/535#discussion_r1973610517) we have decided on using `Date.getTime()`.